src/Microsoft.ML.LightGBM/LightGbmMulticlassTrainer.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.Runtime.LightGBM;
+
+[assembly: LoadableClass(LightGbmMulticlassTrainer.Summary, typeof(LightGbmMulticlassTrainer), typeof(LightGbmArguments),
+    new[] { typeof(SignatureMultiClassClassifierTrainer), typeof(SignatureTrainer) },
+    "LightGBM Multi-class Classifier", LightGbmMulticlassTrainer.LoadNameValue, LightGbmMulticlassTrainer.ShortName, DocName = "trainer/LightGBM.md")]
+
+namespace Microsoft.ML.Runtime.LightGBM
+{
+
+    public sealed class LightGbmMulticlassTrainer : LightGbmTrainerBase<VBuffer<float>, OvaPredictor>
```

Comment:
do we need this line?

Reply:
yes

---
In reply to: [197550818](https://github.com/dotnet/machinelearning/pull/392#discussion_r197550818) [](ancestors = 197550818)